src/test/java/io/github/christophermanahan/carnitas/RouterTest.java

```diff
@@ -5,82 +5,100 @@
 import java.util.List;
 import java.util.function.Function;
 
-import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class RouterTest {
     @Test
     void itProcessesAGETRequestIntoAResponseIfTheRouteHasBeenAdded() {
-        Function<HTTPRequest, HTTPResponse> handler = (HTTPRequest request) -> new HTTPResponse(HTTPResponse.Status.OK);
+        Function<HTTPRequest, HTTPResponse> handler = (HTTPRequest request) -> new ResponseBuilder()
+          .setStatus(HTTPResponse.Status.OK)
+          .get();
         HTTPRequest request = new HTTPRequest(HTTPRequest.Method.GET, "/simple_get");
         Router router = new Router()
           .get("/simple_get", handler);
 
         HTTPResponse response = router.handle(request);
 
-        assertArrayEquals(handler.apply(request).serialize(), response.serialize());
+        assertTrue(handler.apply(request).equals(response));
```

Comment:
This is another occassion where I would use `assertEquals`
```suggestion
        assertEquals(response, handler.apply(request));
```